.github/actions/setup/action.yaml

```diff
@@ -4,7 +4,7 @@ inputs:
   node-version:
     required: false
     description: The version of Node to use.
-    default: '18.x'
+    default: '22.x'
```

Comment:
It should not make much difference, but I had been keeping this at the same level as the CSpell support level, which is 18+.

Reply:
I should put 18.x in the .tool-versions file in that case? I can even configure the action to take it from there.

Reply:
Yes, that would be great.